core/src/main/java/bio/terra/pearl/core/service/survey/SurveyParseUtils.java

```diff
@@ -134,6 +162,14 @@ public static SurveyQuestionDefinition unmarshalSurveyQuestion(
         return definition;
     }
 
+    private static String nodeToString(JsonNode node) {
+        if (node.getNodeType().equals(JsonNodeType.STRING)) {
+            return node.asText();
+        } else {
+            return node.toString();
+        }
+    }
+
```

Comment:
another drive by fix - most of our survey question definitions were actually blank because if you pass a json object through the normal `asText`  it will just be "" since it isn't 'text'. This was affecting all questions that were translated - now, the name will be, e.g. `{ "en": "Title", "es": ... `